File: scenarios/multi_step_agent/fixtures/render_summary.py
from __future__ import annotations

import json
import os


def main() -> None:
    normalized = json.loads(os.environ["NORMALIZED_JSON"])
    print(
        json.dumps(
            {
                "summary": {
                    "status": "success",
                    "items": normalized["count"]
                },
                "evidence_count": normalized["count"],
                "evidence": normalized["items"]
            }
        )
    )


if __name__ == "__main__":
    main()

File: scenarios/multi_step_agent/fixtures/render_summary.ts
const normalized = JSON.parse(process.env.NORMALIZED_JSON ?? "{}") as { count: number; items: string[] };

console.log(
  JSON.stringify({
    summary: {
      status: "success",
      items: normalized.count,
    },
    evidence_count: normalized.count,
    evidence: normalized.items,
  }),
);
